doc/antora/modules/developers/pages/module_interface.adoc

```diff
@@ -76,27 +76,18 @@ different stages. The components are:
 
 [width="100%",cols="35%,65%",options="header",]
 |===
-|Component |Description
-|authorization |Check that a user exists, decide on an authentication
-method or +
-proxy realm, and possibly apply some attributes to be returned +
-in the reply packet.
-
-|authentication |Verify that the password is correct.
-
-|preaccounting |Decide whether to proxy the request, and possibly add
-attributes +
-that should be included in any logs.
-
-|accounting |Record the request in the log.
-
-|checksimul |Count the number of active sessions for the user.
-
-|postauth |Process the response before it’s sent to the NAS.
-
-|preproxy |Process a request before it’s proxied.
-
-|postproxy |Filter attributes from a reply to a proxied request.
+| Component      | Description
```

Comment:
There isn't a lot of point in reformatting this when it's entirely wrong for v4.